tools/benchmark/cmd/root.go

```diff
@@ -67,6 +67,7 @@ func init() {
 	RootCmd.PersistentFlags().StringVar(&tls.CertFile, "cert", "", "identify HTTPS client using this SSL certificate file")
 	RootCmd.PersistentFlags().StringVar(&tls.KeyFile, "key", "", "identify HTTPS client using this SSL key file")
 	RootCmd.PersistentFlags().StringVar(&tls.TrustedCAFile, "cacert", "", "verify certificates of HTTPS-enabled servers using this CA bundle")
+	RootCmd.PersistentFlags().BoolVar(&tls.InsecureSkipVerify, "insecure-skip-tls-verify", false, "skip server certificate verification")
```

Comment:
Firstly, i want use "insecure-skip-verify". But i found the name in [etcdctl](https://github.com/etcd-io/etcd/blob/main/etcdctl/ctlv3/ctl.go#L69) is "insecure-skip-tls-verify".